app/modules/entities/components/lib/works_browser_helpers.js

```diff
@@ -3,18 +3,21 @@ import properties from '#entities/lib/properties'
 import { I18n } from '#user/lib/i18n'
 import { intersection, pluck, uniq } from 'underscore'
 
-export async function getWorksFacets ({ works, context }) {
+export function getWorksFacets ({ works, context }) {
   const contextProperties = facetsProperties[context]
   const { facets, facetsSelectedValues } = initialize({ contextProperties })
 
   const valuesUris = []
   works.forEach(setWorkFacets({ facets, valuesUris, contextProperties }))
+  removeFacetsWithNoKnownValue(facets)
+  return { valuesUris, facets, facetsSelectedValues }
+}
 
+export async function getFacetsSelectors (facetsObj) {
```

Comment:
I think the function name could be more indicative of how it works with the object returned by `getWorksFacets`, and how it's different from it
```suggestion
export async function getWorksFacetsBasicInfoAndSelectors (worksFacets) {
```
maybe?

Reply:
knowing #383 introduce more than works type in facets (ie humans and editions) i would like to keep it entity type agnostic, but informing that this function is fetching server side entities totally make sense. I propose `getFacetsEntitiesBasicInfo` in commit 14cb936f3
 
your second suggestion `worksFacets` doesnt reflect enough for me what it is: a custom `Object` having very specific keys, but not the facets themselves.  `worksFacets` sounds like this variable is facets, but its not.